TU20Bot/Configuration/Controllers/AuthenticationController.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using EmbedIO;
+using EmbedIO.Routing;
+using EmbedIO.WebApi;
+using Jose;
+using MongoDB.Driver;
+using TU20Bot.Models;
+
+namespace TU20Bot.Configuration.Controllers {
+    public class AuthenticationController : ServerController {
+
+        [Route(HttpVerbs.Post, "/login")]
+        public string Login([QueryField] string username, [QueryField] string password) {
+
```

Comment:
This is the definition of a nitpick but these little blank lines at the start of function definitions drive me crazy!!